woocommerce/woocommerce
# Jetpack Changelogger: Add PR number to entries

### All Submissions:






### Changes proposed in this Pull Request:



When creating a pull request along with a changelog entry, the PR number isn't available yet. Instead of retroactively adding it by making sure to create a PR before submitting a changelog entry, this PR appends the number automatically.

### How to test the changes in this Pull Request:

1. Create a new branch from this one locally
2. Add a changelog entry to WooCommerce without adding a pull request number (because you haven't create a PR yet)

```
pnpm nx changelog woocommerce
```
3. Commit the changelog and push to Github
4. Create a draft pull request
5. Check the logs on the `Run lint checks potentially affecting projects across the monorepo` Github Action and see that Matticbot will append the changelog entry with a markdown link to the pull request.
6. Commit a random change to WooCommerce and see the Action run again, this time without amending the changelog.

### Other information:




### FOR PR REVIEWER ONLY:
